src/terms/nonlocal.jl

```diff
@@ -182,13 +180,15 @@ function build_projection_vectors_(basis::PlaneWaveBasis, atoms, kpt::Kpoint)
             # k+G in this formula can also be G, this only changes an unimportant phase factor
             structure_factors = [cis(-2T(π)*dot(kpt.coordinate + G, r)) for G in G_vectors(kpt)]
             for iproj = 1:count_n_proj_(psp)
-                proj_vectors[:, offset+iproj] = structure_factors .* form_factors[:, iproj]
+                @views proj_vectors[:, offset+iproj] .= structure_factors .*
+                                                        form_factors[:, iproj] ./ 
+                                                        sqrt(model.unit_cell_volume)
```

Comment:
```
                @views proj_vectors[:, offset+iproj] .= (structure_factors
                                                         .* form_factors[:, iproj] 
                                                         ./ sqrt(model.unit_cell_volume))
```
Also why not put the `@views` in front of the `for`? The line above also does a copy.

Reply:
I don't think so?

Reply:
ah sorry ... stupid me ... that's the line you removed. All is well ;). I think for such cases I would actually write `.* @view form_factors[:, iproj]` to avoid the deep indention, but I'm happy to leave it as is.